### ISSUE TITLE
warm_up training for renderer

### ISSUE THREAD

**Issue by mengxuyiGit** [syguan96/NeuroFluid#2]
Hi, as stated in paper section 3.3.1, there is a warming up for renderer. However, I cannot find this in the training code. And the update of lr across training steps is also different between code and paper. 
Did I miss the correct place of code? Or is the code updated after paper?

Many thanks!

**Comment by syguan96** [comment]
Hi, thanks for your interest! I have uploaded the code for warmup training ([see this](https://github.com/syguan96/NeuroFluid/blob/main/train_renderer.py)). For the training configs, please stay tuned for the final version.

**Comment by mengxuyiGit** [comment]
Thanks for instant reply! 
So can I use this train_renderer with the generated data to train renderer stand alone ?

**Comment by mengxuyiGit** [comment]
i would like to train the renderer alone, may i have a file containing "particle_pos" of bunny? thanks!

**Comment by syguan96** [comment]
Here are the images and particles at T=0. 

[git.zip](https://github.com/syguan96/NeuroFluid/files/8937517/git.zip)

**Comment by mengxuyiGit** [comment]
Thanks very much!
By the way, when I train using the default 4 views settign on 1 GPU, it will give OOM error. The num_views in the trainer_renderer loops can at most support 2 views. 3 views will already take up 10G. May I know the requirement on the GPU? What GPU are you using?

Many thanks!

**Comment by syguan96** [comment]
I use one 3090 GPU
